core/src/main/java/greencity/service/impl/FavoritePlaceServiceImpl.java

```diff
@@ -30,6 +31,7 @@
     private UserService userService;
     private PlaceService placeService;
     FavoritePlaceDtoMapper favoritePlaceDtoMapper;
+    FavoritePlaceMapper favoritePlaceMapper;
```

Comment:
You need to inject only ModelMapper - not several mapper and use it as modelMapper.map(). And based on the classes you are passing - modelmapper should decide what converter to use.

Reply:
In order to follow the unified approach currently custom mappers extend AbstractConverter and need to override convert method to be further used. All of them were added to MapperConfig class.

Reply:
Redundant mappers were removed, service class and respective test were refactored using modelMapper.map().